extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/NettyProcessor.java

```diff
@@ -243,4 +246,12 @@ RuntimeInitializedClassBuildItem runtimeInitBcryptUtil() {
         // this holds a direct allocated byte buffer that needs to be initialised at run time
         return new RuntimeInitializedClassBuildItem(EmptyByteBufStub.class.getName());
     }
+
+    //if debug logging is enabled netty logs lots of exceptions
+    //see https://github.com/quarkusio/quarkus/issues/5213
+    @BuildStep
+    LogCleanupFilterBuildItem cleanup() {
+        return new LogCleanupFilterBuildItem(PlatformDependent.class.getName() + "0", Level.TRACE, "direct buffer constructor",
+                "jdk.internal.misc.Unsafe");
```

Comment:
Should we include `sun.misc.Unsafe` too (I see some log messages in that class with this string)?

Reply:
Done